### PR TITLE
[Documentation:Developer] Ansible Create Term & Course Docs

### PR DESCRIPTION
**Pull request by ethanfordwork**
The Ansible Create Course and Create Term Documentation. This will allow for user to create a term and a course on their web server following an ansible install.

### CODE REVIEW

**Review by IDzyre — APPROVED** [review]
Once the tests pass, its good to merge

File: _docs/sysadmin/configuration/ansible_course_creation.md [hunk 1]
```diff
@@ -0,0 +1,36 @@
+---
+title: Course Creation
+category: System Administrator > Configuration & Administration
+redirect_from:
```
Comment (bmcutler) [inline _docs/sysadmin/configuration/ansible_course_creation.md:4]: The redirect is only needed when the page used to live under a different path (for backwards compabtibility).  Please remove these lines from the new files

File: _docs/sysadmin/configuration/course_creation.md [hunk 2]
```diff
@@ -5,6 +5,9 @@ redirect_from:
   - /sysadmin/course_creation
 ---
 
+**Note**: If you are using Ansible for managing Submitty, 
+you can follow these [instructions](ansible_course_creation) to create a course using Ansible.
```
Comment (bmcutler) [inline _docs/sysadmin/configuration/course_creation.md:9]: you may need to specify the full path here (if it was relying on the shorter version that I've asked you to remove)

**Review by bmcutler — CHANGES_REQUESTED** [review]
see notes

**Comment by bmcutler** [comment]
Also, I have tried to put every webpage into the index, that is, please edit:
navtreedata.js

File: _docs/sysadmin/configuration/ansible_course_creation.md (hunk 1 above)
Comment (ethanfordwork) [reply _docs/sysadmin/configuration/ansible_course_creation.md]: I removed the redirect line from the new file, ansible create course and term documentation.

File: _docs/sysadmin/configuration/course_creation.md (hunk 2 above)
Comment (ethanfordwork) [reply _docs/sysadmin/configuration/course_creation.md:9]: I comitted and it seems that test still accepts the current specified path, so no need to input the entire path.

**Comment by ethanfordwork** [comment]
On my most recent commit, I added the ansible documentation to the navtreedata.js file. @bmcutler

**Review by bmcutler — APPROVED** [review]

**Review by bmcutler — APPROVED** [review]

**Pull request merged**